src/Console/Commands/MakeWebSSO.php

```diff
@@ -19,6 +19,13 @@ public function handle()
         // is an expedient way of writing this cleanly!
         Artisan::call(MakeDuo::class);
 
+        // Generate the migration unless they have it explicitly turned off in the .env, we default to DB logging + file logging.
+        print("Add SSO_DB_LOG_ENABLED=false to your .env turn off database request logging \n");
+        if (env('SSO_DB_LOG_ENABLED',true) != false) {
```

Comment:
So this is a non-obvious Laravel-ism -- you should only use the `env()` helper inside a config file. If somebody runs [the `config:cache` command, `env('ANYTHING')` will always return null](https://laravel.com/docs/6.x/configuration#configuration-caching).

You can add the config option for this to [`config/nusoa.php`](https://github.com/NIT-Administrative-Systems/SysDev-laravel-soa/blob/master/config/nusoa.php) and reference it [with the config() helper](https://laravel.com/docs/6.x/configuration#accessing-configuration-values) instead. Then it plays nice with cached configs.